src/qt/bitcoin.h

```diff
@@ -67,8 +67,10 @@ class BitcoinApplication: public QApplication
     void parameterSetup();
     /// Create options model
     void createOptionsModel(bool resetSettings);
-    /// Update prune value
-    void SetPrune(bool prune);
+    /// Initialize prune setting. If prune is set, intentionally override
+    /// existing prune size with the default size since this is called when
+    /// choosing a new datadir.
```

Comment:
In commit "qt: Rename SetPrune() to InitializePruneSetting()" (192151ff193ceff8ad7177b07f0b98bf0ebcfc0d)

I think this comment would be a better placed inside the `InitializePruneSetting` method body because it's describing and justifying the implementation of the function, which (I think) is pretty confusing without a nearby explanation. I also think it would be better to add this comment in the earlier commit "qt: Force set nPruneSize in QSettings after intro" (6928c50fa3104ce1b77c5d12928d49584fa536d5) instead of here, because that is the commit which introduces the behavior this comment is describing.

Reply:
Done.